_posts/2025-04-03-vegastrike-0.9.0-release.md

```diff
@@ -0,0 +1,73 @@
+focused around restoring support for Mac and Windows. It is therefore with great pleasure that the Vega Strike Development
+Team is pleased to announce that we have been able to not only restore providing builds for all three of our major platforms
+(Linux, Mac, and Windows) but we have also been able to provide installers for Mac and Windows. A great thanks goes out
+to Stephen G. Tuggy and others that have contributed and tested on these platforms to help make this great achievement
+possible.
+
+As noted with the 0.8.0 release we have completely dropped support for Python2 and are 100% Python3 oriented. We have also
+dropped providing builds with the LEGACY OpenGL support; however, support remains in the source code should anyone still
+need it.
+
+As part of updating the Vega Strike Engine we have started a journey to update how we store and load data. With the 0.9.0
+release we have moved our a number of our data assets from defaulting to using a CSV format to now using a JSON format.
+While the format is not as nice to use in tools like LibreOffice Calc or Microsoft Excel, this will help reduce errors and
+make it easier to maintain the files.
+
+Finally, our Vega Strike Engine API has been updated to Engine API 2 which moves some functionality to the Game Assets in
+addition to the CSV to JSON file format changes.
+
+As a reminder, the Vega Strike Engine application is available as `vegastrike-engine` which requires the `-D` parameter to specify
+where to find the Game Assets. The `vegastrike` binary is still being provided at this time and it will continue to autofind the
+the assets like with previous versions. However, we strongly advise moving to the new `vegastrike-engine` binary and specifying
+the parameter for the Game Assets as we will be dropping it in a future release.
+
+For a complete list of differences see [Vega Strike Engine Delta][vsedelta] and [Vega Strike: Upon the Coldest Sea Delta][vsutcsdelta].
+
+## Want to help out?
+
+There's a lot to do. We're working diligently to update Vega Strike. Version 0.10.0 is our next step along the line which will be
+continuing to bring big improvements to the engine. We've already got several releases planned in the process of continuing to update
+our dependencies and fix bugs; you can see our [Road Map][roadmap]. We need everything from coding, story development, art work,
+documentation, and even just generally helping the community.
+
+The development team is active at [Gitter.im][gitter] and we'd love to hear from you.
+
+You can also find the community at the [Vega Strike Forums][forums].
+
+## Where can I find the release?
+
+[Vega Strike Engine 0.9.0][vse]
+[Vega Strike: Upon the Coldest Sea 0.9.0][vsutcs]
+
+NOTE: Presently just the Linux builds are provided. The Vega Strike Team is still working to restore Mac and Windows support.
+
+[sf]: https://sourceforge.net/projects/vegastrike/
```

Comment:
Can probably omit this as well. As far as I can tell, this link isn't actually used anywhere.

Reply:
yeah, I think I had it in there prior for the 0.5.3 releases that are still hosted there which I'm no longer referencing since we now have Windows/Mac support

Reply:
removed